group_vars/all

```diff
@@ -1,5 +1,7 @@
 ######## Unity vars ########
 
+host: 192.168.70.217
+host2: 192.168.70.218
 unity_host: 192.168.70.217
 unity_host2: 192.168.70.218
```

Comment:
Let's remove "host" or "unity_host"

Reply:
So you should remove all unused vars and add unity_ip like I did it in wiki

Reply:
So, if i understand correctly, you want to change group_vars to unity_ip and dont' want to change playbooks. You do understand that it will ruin the work of playbooks, right?

Reply:
I do use both host and unity_host parameters in the playbooks, so they are not unused.

Reply:
I do